pkg/apis/app/v1alpha1/workspace_types.go

```diff
@@ -17,6 +17,20 @@ type Module struct {
 	Version string `json:"version"`
 }
 
+// VCS holds all the information needed to connect the workspace to a VCS repository
+type VCS struct {
+	// Token ID of the VCS Connection (OAuth Connection Token) to use
+	TokenID string `json:"token_id"`
```

Comment:
Would it be possible to fetch this token from the secret named `sensitivevars` or from the secret named `workspacesecrets`? Having an API token of any kind inside of a CustomResource is going to be a security issue for DevOps/SRE teams.

Reply:
The `TokenID` is the ID of the OAuth token created in TFC. It can be found here: `https://app.terraform.io/app/<your org>/settings/version-control`. 

I will update the comments to make it clearer.